support-frontend/assets/pages/paper-subscription-landing/components/hero/discountCopy.jsx

```diff
@@ -5,20 +5,27 @@ import { type Node } from 'react';
 import { flashSaleIsActive, getSaleCopy } from 'helpers/flashSale';
 import { GBPCountries } from 'helpers/internationalisation/countryGroup';
 
-type DiscountCopy = {
+export type DiscountCopy = {
   roundel: string[],
   heading: string | Node,
 };
 
-const discountCopy: DiscountCopy = {
-  roundel: ['Save up to', '37%', 'a year'],
-  heading: 'Save up to 37% a year with a subscription',
-};
+const discountCopy = (discountPercentage: number): DiscountCopy => (
+  discountPercentage > 0 ?
+    {
+      roundel: ['Save up to', `${discountPercentage}%`, 'a year'],
+      heading: `Save up to ${discountPercentage}% a year with a subscription`,
+    } :
+    {
+      roundel: [],
+      heading: `Save money with a subscription`,
+    }
```

Comment:
This default is for when the page is being 'server side rendered' during the build process because at that point the pricing information is not available.